src/store/SubjectStore.js

```diff
@@ -0,0 +1,44 @@
+import { types, flow } from 'mobx-state-tree'
+import { subjects } from '@zooniverse/panoptes-js'
+import ASYNC_STATES from 'helpers/asyncStates'
+
+const TEMPORARY_SUBJECT_ID = '72815'
+
+const Subject = types
+  .model('Subject', {
+    id: types.optional(types.string, ''),
+    locations: types.array(types.frozen({})),
+    metadata: types.frozen({}),
+  })
+
+const SubjectStore = types.model('SubjectStore', {
+  asyncState: types.optional(types.string, ASYNC_STATES.IDLE),
+  current: types.optional(Subject, {}),
+  index: types.optional(types.number, 0),
+}).actions(self => ({
+  changeIndex: (index) => {
+    self.index = index
+  },
+
+  fetchSubject: flow (function * fetchSubject (id) {
+    self.asyncState = ASYNC_STATES.FETCHING
+    try {
+      const response = yield subjects.get({ id: TEMPORARY_SUBJECT_ID })
+      if (response.body.subjects[0]) {
```

Comment:
I assume at this stage of development, we're not worried about creating a Subject queue yet. This is fine 👌

Reply:
No. I'm not sure a subject queue would be necessary since an editor always has to select a specific subject to edit from an index.

Reply:
Wait, goshdangit - you're completely right, Will. This specific project does NOT benefit from having a Subject queue. Please ignore my earlier comment, my brain was defaulting to "standard Zooniverse Classifier behaviour" while reading this code, instead of properly addressing the purpose of this project's Subject Store.